### DISCUSSION TITLE
What is the simplest math problem that no one can solve ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#78] [Galaxy Qs] [answered]
What is the simplest math problem that no one can solve ?

**Comment by mitchiemt11** [comment]
🤔

**Comment by mitchiemt11** [comment, marked as answer]
3x + 1
The Collatz conjecture

**Reply by Sriansh-raj** [reply]
correct 🎉

**Reply by Sriansh-raj** [reply]
Congratulations @mitchiemt11 you just got the `Galaxy Brain🥉` achievement. 🎉

Don't forget to create a PR and update your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to create a history 😃
